Introduction_To_XenServer_Fuel_Plugin.md

```diff
@@ -1,6 +1,6 @@
-### Introduction To XenServer Fuel Plugin
+### Introduction to XenServer Fuel Plugin
 
-As becoming part of the Big Tent, Mirantis Fuel has already made itself one of the leader installers for OpenStack and offers an pluggable architecture that enable you to add new capabilities to your environments. To take advantage of that, XenServer Fuel Plugin is aiming to enables use of the XenServer open source hypervisor (version 6.5. SP1) as a compute provider on Mirantis OpenStack, with commercial support options from Citrix. To be more specific we want to achieve the following major goals:
+As becoming part of the Big Tent, Mirantis Fuel has already made itself one of the leader installers for OpenStack and offers an pluggable architecture that enable you to add new capabilities to your environments. XenServer Fuel Plugin is aiming to enables use of the XenServer open source hypervisor (version 6.5. SP1) as a compute provider on Mirantis OpenStack, with commercial support options from Citrix. This blog post will cover how to do the following topics with XenServer Fuel Plugin:
```

Comment:
The previous language was better; "To be more specific, we want to achieve the following major goals".
The proposed language of "how to do the following topics" is much less precise.